src/_P076_HLW8012.ino

```diff
@@ -482,13 +492,13 @@ boolean Plugin_076(uint8_t function, struct EventStruct *event, String& string)
           # if PLUGIN_076_DEBUG
           addLogMove(LOG_LEVEL_INFO,
                      strformat(F("P076: Read values - V=%.2f - A=%.2f - W=%.2f - VAR=%.2f - VA=%.2f - Pf%%=%.2f - Ws=%.2f"),
-                               p076_hvoltage,
-                               p076_hcurrent,
-                               p076_hpowerActive,
-                               p076_hpowerReactive,
-                               p076_hpowerApparent,
-                               p076_hpowfact,
-                               p076_henergy));
+                               HLW[0],
+                               HLW[1],
+                               HLW[2],
+                               HLW[3],
+                               HLW[4],
+                               HLW[5],
+                               HLW[6]));
```

Comment:
You use `HLW[x]` everywhere, but it is hard to read.
Can you make defines for these indices? (and then also use them when assigning in the lines above)

```c++
#define P076_VOLT  0
#define P076_CURR 1
....
```
And then later `HLW[P076_VOLT]` instead of `HLW[0]`

Reply:
Good point. I've changed it.